routing/probability_bimodal.go

```diff
@@ -301,6 +301,28 @@ func (p *BimodalEstimator) calculateProbability(directProbability float64,
 		return directProbability
 	}
 
+	// If we have up-to-date information about the channel we want to use,
+	// i.e. the info stems from results no longer ago than the decay time,
```

Comment:
```suggestion
	// i.e. the info stems from results no longer than the decay time,
```

Reply:
Hm, I think usage of `ago` is correct here